src/modules/components/proposal/proposalActions/proposalActions/proposalActions.stories.tsx

```diff
@@ -35,6 +39,27 @@ export const TokenWithdraw: Story = {
     },
 };
 
+/**
```

Comment:
As mentioned on Asam's PR as well, I'd avoid keep updating the `ProposalActions` stories for every new action but simply create a story file under the `/actions` folder